settings.gradle

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  */
 
+['fiat', 'keiko', 'kork'].each { prj ->
```

Comment:
❤️  maybe worth adding a note here (similar to the one you had in the description of the PR) or `gradle.properties` (or readme) prob gradle.props is the most likely to get noticed 

> a command line flag, e.g. -PkorkComposite=true
> a project property via gradle.properties
> a global project property via ~/.gradle/gradle.properties

Reply:
good idea, will do!

Reply:
I have a feeling bumpdeps won't preserve comments in`gradle.properties`, but I don't know for sure...

Reply:
ahh yeah you are correct..

I will move this into the README for now

we could always solve bumpdeps with a regex instead of using java.util.Properties, what do you think about that?

Reply:
Yeah, seems reasonable.

Reply:
https://github.com/spinnaker/bumpdeps/pull/6/files